02_array_shift/array_shift.test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const array_shift = require('./array_shift.js');
+
+test('should return an array with the elements in reversed order', () => {
+  let actual = array_shift.insertShiftArray([2, 4, 6, 8], 5);
+  let expectedValue = [2, 4, 5, 6, 8];
+  expect(actual).toEqual(expectedValue);
+
+  actual = array_shift.insertShiftArray([4, 8, 15, 23, 42], 16),
+  expectedValue = [4, 8, 15, 16, 23, 42];
+  expect(actual).toEqual(expectedValue);

```

Comment:
This `expect` appears to be testing the same thing as the first `expect`. We do not need duplicate tests that test the same condition.